recipes/master.rb

```diff
@@ -77,12 +76,11 @@
       action :modify
     end
 
-    include_recipe 'krb5_utils'
```

Comment:
They are idempotent. They also, cannot really be changed without the rest of the initialization being re-run, to get the proper owners on directories. Unfortunately, our current checks only check existence of files/directories in HDFS, versus full permissions. I have been hoping to fix that with HDFS LWRPs in the Hadoop cookbook, but it's not been a huge priority. If someone changes their keytabs/principals, they're probably going to have a bad time, either way.